### ISSUE TITLE
Ability to disable or secure/lockdown some commands

### ISSUE THREAD

**Issue by GoogleCodeExporter** [baluragala/gtalksms#209]
```
Ability to remove or "secure"/"lockdown" certain commands.

e.g. Disable, delsms:all or require a password or PIN

Great work!
```

Original issue reported on code.google.com by `thekapsl...@gmail.com` on 2 Aug 2011 at 2:03

**Comment by GoogleCodeExporter** [comment]
Original comment by `Florent....@gmail.com` on 2 Aug 2011 at 12:08
- Changed title: **Enhancement: Ability to disable or secure/lockdown some commands**
- Added labels: **Type-Enhancement**
- Removed labels: **Type-Defect**

**Comment by GoogleCodeExporter** [comment]
Original comment by `Florent....@gmail.com` on 2 Aug 2011 at 12:09
- Changed title: **[ER] Ability to disable or secure/lockdown some commands**

**Comment by GoogleCodeExporter** [comment]
Original comment by `fschm...@gmail.com` on 31 Aug 2011 at 10:46
- Changed title: **Ability to disable or secure/lockdown some commands**

**Comment by GoogleCodeExporter** [comment]
Original comment by `fschm...@gmail.com` on 1 Dec 2011 at 5:36
- Removed labels: **Priority-Medium**

**Comment by GoogleCodeExporter** [comment]
Original comment by `Florent....@gmail.com` on 7 Jan 2012 at 9:01
- Changed state: **Started**

**Comment by GoogleCodeExporter** [comment]
Original comment by `Florent....@gmail.com` on 7 Jan 2012 at 9:04

**Comment by GoogleCodeExporter** [comment]
```
A new panel, Command Manager, allows to disable/enable a whole command.
By instance delsms could be disabled.

Disable a specific subcommand needs more refactoring.
```

Original comment by `Florent....@gmail.com` on 9 Jan 2012 at 12:12
- Changed state: **Fixed**

**Comment by GoogleCodeExporter** [comment]
```
Beta 3.4.1 available in the download section.
```

Original comment by `Florent....@gmail.com` on 16 Feb 2012 at 10:58

**Comment by GoogleCodeExporter** [comment]
```
Fixed in version 3.5
```

Original comment by `Florent....@gmail.com` on 2 Apr 2012 at 9:38
- Changed state: **Deployed**
